src/features/search/sections/tests/DateHour.native.test.tsx

```diff
@@ -41,9 +41,9 @@ describe('DateHour component', () => {
       },
     })
 
-    const { queryByText } = render(<DateHour />)
+    const { getByText } = render(<DateHour />)
 
-    expect(queryByText('le Jeudi 3 mars 2022')).toBeDefined()
+    expect(getByText('le Jeudi 3 mars 2022'))
```

Comment:
ça te génère un code smell, du coup je te suggères :
`expect(getByText('le Jeudi 3 mars 2022')).toBeTruthy()`

Reply:
Hm, en soi le `expect` ne sert à rien vu que `getByText` throw s'il ne trouve rien. C'est d'ailleurs pour ça que j'ai utilisé `getByText` et non `queryByText`.

Mais ouaip OK alors :D

Reply:
Après coup, je me dis que j'aurai dû faire 

```suggestion
    expect(() => {
      getByText('le Jeudi 3 mars 2022')
    }).not.toThrow()
```
mais ça fait beaucoup de code pour la même chose...

Reply:
Je ne crois pas l'avoir trop vu/utilisé mais pourquoi pas, ça peut être une autre façon de l'utiliser.
Après tu aurais pu faire un `it.each` avec 2 paramètres (ton option et ton texte attendu) pour avoir un peu moins de lignes de code mais ça le fait comme ça :)

Reply:
Oui c'est vrai !
Je n'ai pratiquement jamais utilisé `it.each`. Sûrement parce que j'écris mes tests un à un.
Du coup je ne sais pas l'utiliser, mais j'ai vu que toi tu l'utilisais beaucoup :D

Je vais tester pour voir :)